src/main/java/wrup/spring/a_wstrzykiwanie_zależności/silnik/OlejNapędowy.java

```diff
@@ -0,0 +1,13 @@
+package wrup.spring.a_wstrzykiwanie_zależności.silnik;
+
+/**
+ * Ziarno spingowe wstrzykiwane do klasy Samochód {@link wrup.spring.a_wstrzykiwanie_zależności.Samochód Samochód}
```

Comment:
Zwykle link się używa po to, by przeskoczyć. Nieco dziwnie będzie się czytać Samochód _Samochód_, gdzie drugie jest linkiem. Proponuję pierwsze Samochód usunąć.